ui/shared/utils/constants.js

```diff
@@ -1316,38 +1316,70 @@ export const SV_IN_SILICO_GROUP = 'Structural'
 export const NO_SV_IN_SILICO_GROUPS = [MISSENSE_IN_SILICO_GROUP, CODING_IN_SILICO_GROUP]
 export const SPLICE_AI_FIELD = 'splice_ai'
 
+const PRED_COLOR_MAP = ['green', '#90ee90', 'grey', 'yellow', 'red', '#8b0000']
+
+export const getPredictColor = (value, pathHigher, thresholds) => {
+  let colorIndex = 0
+  const v = pathHigher ? value : -1 * value
```

Comment:
all of the `pathHigher` logic seems to have been added for scores that we have as text scores in the data in seqr, not as numeric scores. Once you take out all the thresholds for scores that did not used to have thresholds and can't have them, this logic should be unneeded

Reply:
You are right. SIFT and FATHMM have lower pathogenic scores. They both have non-numeric scores.